Sources/CryptoSwift/BlockDecryptor.swift

```diff
@@ -87,7 +87,7 @@ public class BlockDecryptor: Cryptor, Updatable {
 
   public func seek(to position: Int) throws {
     guard var worker = self.worker as? SeekableModeWorker else {
-      fatalError("Not supported")
+      print("Not supported")
```

Comment:
The correct other way to handle it is to throw a custom exception so that you can handle the exception as you want. But for people who cares about this error, removing it with a print won't help them, because this error has a meaning that cannot be ignored for everyone.

Reply:
totally. bo fatal error. It can be used ad a weakness to decrypt the original message (oracle attack kindof)